keyboards/pimoroni/keybow2040/readme.md

```diff
@@ -0,0 +1,101 @@
+Keybow 2040
+===========
+
+The [Pimoroni Keybow 2040][keybow2040] is a 16 key mechanical macropad
+controlled by a Raspberry Pi RP2040, with Kailh hot swap sockets and
+per key RGB lighting, in a decorative FR4 sandwich. The Keybow 2040 is
+also known by its codename Venus Dust, or its product codes of the
+form PIM56x.
+
+* Keyboard Maintainer: [Tony Finch](https://github.com/fanf2)
+* Hardware Supported: Keybow 2040
+* Hardware Availability: [Pimoroni][keybow2040]
+
+[keybow2040]: https://shop.pimoroni.com/products/keybow-2040
+
+Make example for this keyboard (after setting up your build environment):
+
+    make pimoroni/keybow2040:default
+
+Flashing example for this keyboard:
+
+    make pimoroni/keybow2040:default:flash
+
+See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools)
+and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide)
+for more information. Brand new to QMK? Start with our
+[Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
+
+
+bootloader
+----------
+
+The procedure to flash the firmware is similar to other RP2040
+devices. You can enter the bootloader in two ways:
+
+  * **Vulcan nerve pinch**: hold down BOOTSEL while pressing RESET
+    (not so great if you have configured BOOTSEL to do something)
+
+  * **Mr Resetti double-tap**: press RESET twice quickly
+    (only works when running the QMK firmware)
+
+Find the BOOTSEL and RESET buttons on your Keybow 2040 either side of
+the USB socket on the edge of the PCB. When the USB socket is facing
+away from you, BOOTSEL is on the right and RESET is on the left.
+According to the schematic the reset button is called Mr Resetti.
+
+  * _Aside_: The BOOTSEL button disables the QSPI flash that stores
+    the firmware, which makes the RP2040 use its built-in UF2
+    bootloader.
+
+A USB drive will appear called `RPI-RP2`. Copy the firmware file
+`pimoroni_keybow2040_default.uf2` onto the `RPI-RP2` drive using
+whatever software is convenient (e.g. the `cp` command, the macOS
+Finder, or Windows Explorer). The `make :flash` command will copy the
+firmware for you.
+
+The `RPI-RP2` drive will disappear (which might make your OS complain
+that you failed to unmount it first) and a keyboard device will appear
+in its place.
+
+The Pimoroni web site has [instructions for reinstalling CircuitPython
+on the Keybow 2040](https://learn.pimoroni.com/article/circuitpython-and-keybow-2040)
+
+
+notes
+-----
+
+  * The BOOTSEL button doubles as a 17th configurable key.
+
+  * The default keymap has F13 - F20 on the top half of the board
+    (nearest the USB socket), and a navigation cluster (arrows, home,
+    end, page up/down) on the bottom half. The BOOTSEL button changes
+    the RGB matrix mode.
+
+  * The Keybow 2040 CircuitPython firmware uses VID = 0x16D0, PID = 0x08C6,
+    but this QMK firmware has different USB IDs to avoid confusing drivers:
+    VID = 0x5069 ('Pi'), PID = 0x4784 (4 'x' 4)
+
```

Comment:
Not sure how useful this is here. 
```suggestion
```

Reply:
I've left that in because there ought to be somewhere for users to find out what the default keymap is without reading the JSON, and I need somewhere to explain the info.json configuration choices. JSON isn't supposed to have comments, so 🤷